open_schools_platform/organization_management/organizations/views.py

```diff
@@ -277,26 +276,26 @@ def get(self, request, pk):
         return Response({"analytics": AnalyticsSerializer(count_queries_by_statuses(queries)).data}, status=200)
 
 
-class QueriesCirclesOrganizationStudent(ApiAuthMixin, ListAPIView):
+class QueriesOrganizationStudent(ApiAuthMixin, ListAPIView):
```

Comment:
I think OrganizationStudentProfileQueriesApi is better

Reply:
ok